src/api/app/helpers/webui/group_helper.rb

```diff
@@ -0,0 +1,14 @@
+module Webui::GroupHelper
+  def group_management_label
+    if User.current.is_admin?
+      link_to('Group Management', groups_path)
+    else
+      'Group Management'
+    end
+  end
+
+  def group_member_link(group, user)
+    group_maintainer_label = ' (Group maintainer)' if GroupMaintainer.where(group: group, user: user).exists?
```

Comment:
Using models queries in helpers? I think is not the best.

I mean, if you have 30 users to show then you will perform 30 queries to the database, in the other hand if you calculate the maintainers outside the helper only once and passing that Array to this helper then you will only perform one query.

Reply:
Not sure why bullet does not complain about this, but I guess you have a point.
But that means I have to do a couple of changes. I'll drop the commit for now.

Regarding the "models queries". That's exactly what helpers are meant for:-) To abstract logic from views.